tex/main/MarkovChain/closed_form_state_probs/main.tex

```diff
@@ -1,15 +1,28 @@
 \subsection{Graph Theoretical Approach}
+% TODO: Generalise notation in entire section 
 
-This section aims to investigate the connection between Markov chains and graph theory by exploring alternative ways of calculating the state probabilities and considering the problem from different perspectives.
+This section aims to investigate the connection between Markov chains and graph theory by exploring alternative ways of calculating the steady state probabilities and considering the problem from different perspectives.
 
 \subsubsection{Parameters}
-The inputs of the formula are the number of servers \(C\), the threshold \(T\), the system capacity \(N\) and the parking capacity \(M\). 
-Additional parameters of the model are the ambulance arrival rate, the others' arrival rate and the service rate, but for the purpose of this section these will remain unknown (\(\lambda^A, \lambda^o, \mu\)). 
+\begin{itemize}
+    \item the number of servers \(C\),
+    \item the threshold \(T\), 
+    \item the system capacity \(N\),
+    \item the parking capacity \(M\). 
+\end{itemize}
+Additional parameters of the model are the ambulance arrival rate, the others' arrival rate and the service rate (\(\lambda^A, \lambda^o, \mu\)). 
 More specifically, the way these parameters are translated into the model are:
 
 \begin{itemize}
-    \item \textbf{Number of servers (\(C\)):} All service rates \(\mu\) in the Markov chain are multiplied by a coefficient equal to \(v\) for a state \((u,v)\) that stops increasing at \(v=C\). 
-    Thus, the coefficients of the service rate have a lower bound of \(0\) and an upper bound of \(C\).
+    \item \textbf{Number of servers (\(C\)):} All edges in the Markov chain that correspond to a service rate have a weight of: 
```

Comment:
I meant that you would simply point at your definition of the transition rates?

Reply:
Oh okay. So just reference the equation where I defined these transition rates?

Reply:
I think so. I think that's sufficient no. What do you think?

Reply:
I am not sure. I wanted to explain it in terms of a graph theoretic perspective and I felt that I should use different notation to define stuff. But it does make sense to have it that way because it is essentially the same thing,

Reply:
I understand. I'd suggesting solving both problems. I forget your exact notation but let's say 

- `S` is the state space of the Markov chain
- `q_{s_1, s_2}` is the transition rate between `s_1` and `s_2`.
- `w_{e}` is the weight for edge `e`.

Then I suggest we go for something like:

```
We define the weighted digraph \(G=(V, E)\) where \(V=S\) and \((v_1, v_2)\in E\) 
if and only if \(q_{v_1, v_2}>0\). Further more, the weights are given by:

\[
    w_((v_1, v_2)) = q_{v_1, v_}
\]

This way we're rigorously defining the graph using the correct vocabulary but also clearly showing the connection back to the chain?
```

Reply:
Ohhh. I like this. Yeah I will include something like this in.